cogs/games.py

```
@@ -0,0 +1,324 @@
+            
+            return None
+        
+        message = await ctx.send(embed=create_board_embed())
+        
+        number_emojis = ['1️⃣', '2️⃣', '3️⃣', '4️⃣', '5️⃣', '6️⃣', '7️⃣', '8️⃣', '9️⃣']
+        for emoji in number_emojis:
+            await message.add_reaction(emoji)
+        
+        while True:
+            def check(reaction, user):
+                return (user == players[current_player] and 
```

Ruff (0.11.9)

Function definition does not bind loop variable `current_player`

(B023)